### ISSUE TITLE
dmengine.exe removed from bob.jar

### ISSUE THREAD

**Issue by dtandersen** [astrochili/vscode-defold#50]
Looks like the launcher can't find dmengine.exe in 1.9.6.

https://github.com/defold/defold/pull/9724

```
2024-12-28 17:17:49.939 [info] Extracting the vanilla engine from Defold to 'd:\Nextcloud\Projects\defold-dungeon\build\defoldkit\dmengine.exe'
2024-12-28 17:17:49.939 [info] Creating temporary directory: c:\Users\david\AppData\Roaming\Code\User\workspaceStorage\d0911e525a197e133fdd738dd62f94e1\astronachos.defold\tmp
2024-12-28 17:17:49.943 [info] Jar Extracting $: "c:\Users\david\Downloads\Defold-x86_64-win32\Defold\packages\jdk-21.0.5+11\bin\jar" -xf "c:\Users\david\Downloads\Defold-x86_64-win32\Defold\packages\defold-83536c65275c04366b704b2fd3f3d681c88b8267.jar" "_unpack\x86_64-win32\bin\dmengine.exe"
2024-12-28 17:17:49.943 [info] Jar Extracting: [starting]
2024-12-28 17:17:50.172 [info] Jar Extracting: [exit 0]
2024-12-28 17:17:50.172 [info] Copying '_unpack\x86_64-win32\bin\dmengine.exe' to 'd:\Nextcloud\Projects\defold-dungeon\build\defoldkit\dmengine.exe'
2024-12-28 17:17:50.176 [info] Exception occured during copying file: EntryNotFound (FileSystemError): File to move/copy does not exist
2024-12-28 17:17:50.176 [info] Failed to extract the engine executable from Defold'
```

```
'"build\defoldkit\dmengine.exe"' is not recognized as an internal or external command,
operable program or batch file.
```

I found the file in https://github.com/defold/defold/releases/download/1.9.6/bob.jar, but when I copy dmengine.exe to build/defoldkit it gets deleted.

**Comment by astrochili** [comment]
Yes and it’s a bad news for projects without native extensions. Need to find a way to get this exe somewhere…

**Comment by astrochili** [comment]
> I found the file in https://github.com/defold/defold/releases/download/1.9.6/bob.jar, but when I copy dmengine.exe to build/defoldkit it gets deleted.

Defold Kit deletes it before building. A possible workaround is pasting this file to the jar archive of Defold where Defold Kit tries to found it.

**Comment by astrochili** [comment]
Fixed in [2.1.10](https://github.com/astrochili/vscode-defold/releases/tag/2.1.10).

**Comment by dtandersen** [comment]
Confirmed working in 2.1.10!
